core/base/inc/Rtypes.h

```diff
@@ -284,67 +284,75 @@ private:          \
          return ::ROOT::Internal::THashConsistencyHolder<decltype(*this)>::fgHashConsistency;                   \
       }                                                                                                         \
       return false; /* unreacheable */                                                                          \
-   }                                                                                                            \
+   } /** \endcond */                                                                                            \
                                                                                                                 \
 public:                                                                                                         \
-   static Version_t Class_Version() { return id; }                                                              \
-   virtual_keyword TClass *IsA() const overrd { return name::Class(); }                                         \
-   virtual_keyword void ShowMembers(TMemberInspector &insp) const overrd                                        \
+   /** \return Version of this class */ static Version_t Class_Version() { return id; }                         \
+   /** \see ::Class() */ virtual_keyword TClass *IsA() const overrd { return name::Class(); }                   \
```

Comment:
```suggestion
   /** \Return TClass describing current instance */ virtual_keyword TClass *IsA() const overrd { return name::Class(); }                   \
```
The static function`Class` returns the `TClass` describing the current class while the virtual function `IsA` will return the `TClass` describing the actual (most derived) class of the object.